aodn_cloud_optimised/config/dataset/radar_velocity_hourly_averaged_delayed_qc_main.json

```diff
@@ -127,7 +127,7 @@
       "comment": "This value is set on the basis of the offline quality controls applied in the time domain (see abstract)."
     },
     "VCUR_quality_control": {
-      "type": "float",
+      "type": "byte",
```

Comment:
dont change the type for now. This will break the dataset as it will have to be fully reprocessed. I might have to look into it later